addons/base_import_module/models/ir_module.py

```diff
@@ -177,7 +184,7 @@ def _import_module(self, module, path, force=False):
         return True
 
     @api.model
-    def import_zipfile(self, module_file, force=False):
+    def import_zipfile(self, module_file, force=False, with_demo=False):
```

Comment:
can we make it private btw?

Reply:
https://github.com/odoo-dev/odoo/commit/a71b1332b8c980fea94e96a18eb0722092183640